spec/functional/bootstrap_download_spec.rb

```diff
@@ -100,6 +100,14 @@
         run_download_scenario
       end
     end
+
+     context "when provided a custom msi_url with space in path to fetch from" do
+      let(:mock_bootstrap_context) { Chef::Knife::Core::WindowsBootstrapContext.new(
+        { :msi_url => "file:///C:/Program Files/Windows NT/Accessories/wordpad.exe" }, nil, { :knife => {} }) }
```

Comment:
So does this actually download the file? If it does that will make things significantly slower...